tox.ini

```diff
@@ -57,6 +59,7 @@ commands =
     mypy-focal: mypy --python-version 3.7 uaclient/ features/ lib/
     black: black --check --diff uaclient/ features/ lib/ setup.py
     isort: isort --check --diff uaclient/ features/ lib/ setup.py
+    shellcheck: bash -c "shellcheck -S warning tools/{*.sh,make-release,make-tarball} debian/ubuntu-advantage-tools.{config,postinst,postrm,prerm} lib/*.sh sru/release-27.8/*.sh update-motd.d/**"
```

Comment:
This requires us to remember updating the globs on new releases. Maybe it's better to *exclude* what we know we're not going to fix (old sru scripts), something like (note: requires `extglob`);
```
bash -O extglob -c 'shellcheck -S warning tools/{*.sh,make-release,make-tarball} debian/*.{config,postinst,postrm,prerm} lib/*.sh sru/release-!(27|27.3|27.5|27.4|27.4.*|27.7)/*.sh update-motd.d/*'
```
Or we could leave `sru/*` out of shellcheck.

Note that I also made the `debian/` globs more generic and removed the double `*` from `update-motd.d/**` (we're not using `globstar` so I don't think `**` does anything special, but fixme).

Reply:
We also decided to exclude 27.8 sru scripts